specific_use_cases/DFU_work_in_progress/INTRO_annual_consumption_model.ipynb

```diff
@@ -690,12 +724,191 @@
     "hdd_cdd_df.to_csv(filename, index=True)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "459602b9-5898-41c2-99ed-f6d2202afa03",
+   "metadata": {},
+   "source": [
+    "## Step 5: Compute heating degree hours and cooling degree hours\n",
+    "Alternatively, you may be interested in the number of hours in each day that a designated heating or cooling threshold crosses. For Cooling Degree Hours (CDH), this is the number of hours in which the hourly temperature exceeds the cooling degree threshold. Likewise, Heating Degree Hours (HDH) is the number of hours in which the hourly temperature is below the heating degree threshold. We'll use the helper function `compute_hdh_cdh` to calculate HDH and CDH:<br><br>\n",
+    "**CDH = num of hours where (temperature $>$ threshold)<br>\n",
+    "HDH = num of hours where (-1)\\*(temperature $<$ threshold)**<br><br>\n",
+    "We will then display the results for a month to see how trends change throughout the year. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "748022b0-f25a-44aa-9070-a4e4a74d2e9c",
+   "metadata": {},
+   "source": [
+    "### 5a) Compute HDH and CDH\n",
+    "Like the CDD and HDD examples above, we'll use all of the data for our selected DFZ zone to calculate CDH and HDH. Note that we've added an attribute to the data to retain the threshold used to compute the data here. If you forget, look at the attributes of CDH or HDH. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "12f6395e-ee86-46a9-b631-e2495c7af283",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ALL DATA WITHIN DFZ ZONE\n",
+    "data_to_use = data_dfz\n",
+    "num_grid_cells = data_dfz.x.size * data_dfz.y.size # Number of grid cells within the demand forecast region"
+   ]
+  },
+  {
```

Comment:
Neither `data_to_use` nor `num_grid_cells` is used after this point, so I recommend removing this cell as to not confuse people.